app/views/pages/developer-guide/activity-feeds/activity-feeds.liquid

```diff
@@ -10,6 +10,15 @@ An **activity** is a semantic description of an action.
 Get familiar with the [W3C Recommendation for Activity Streams 2.0](https://www.w3.org/TR/2017/REC-activitystreams-core-20170523/). Most examples and part of the implementation you'll find in this guide are based on this specification, that provides a JSON-based syntax that is sufficient to express metadata about activities in a rich, human-friendly but machine-processable and extensible manner. This can include constructing natural-language descriptions or visual representations about the activity, associating actionable information with various types of objects, communicating or recording activity logs, or delegation of potential actions to other applications.
 
 ## Activities
+
+Activities (activity list) is an activity stream and it is an immutable list of immutable objects (activities/events that happened it the past).
+
+- By design you can only add an entry
```

Comment:
If this means that only one entry can be added, we should write it as "By design, you can only add one entry".

Reply:
It means that by design an entry can be added only, update and delete operands are denied. What do you think, do I need to modify this line?